### PR TITLE
doc: use 'index' and 'value' in place of 'i' and 't'

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

**Comment by Gankro** [comment]
This is going to get clobbered by https://github.com/rust-lang/rust/pull/26955

File: src/libcollections/vec_deque.rs [hunk 1]
```diff
@@ -949,8 +949,8 @@ impl<T> VecDeque<T> {
     /// buf.insert(1,11);
     /// assert_eq!(Some(&11), buf.get(1));
     /// ```
-    pub fn insert(&mut self, i: usize, t: T) {
-        assert!(i <= self.len(), "index out of bounds");
+    pub fn insert(&mut self, index: usize, value: T) {
+        assert!(index <= self.len(), "index out of bounds");
```
Comment (Gankro) [inline src/libcollections/vec_deque.rs]: Note that several inner impl docs refer to `i` explicitly (this fn is an example).

**Comment by tshepang** [comment]
now unclobbered @Gankro

**Comment by Gankro** [comment]
@bors r+

Thanks!

**Pull request merged**